test/apps/wboit/app.jsx

```diff
@@ -1,7 +1,6 @@
 import React, {PureComponent, Fragment} from 'react';
 import DeckGL from '@deck.gl/react';
 import {AmbientLight, DirectionalLight, LightingEffect} from '@deck.gl/core';
-import {StaticMap} from 'react-map-gl';
```

Comment:
The RMG dep was upgraded.. is this import meant to be updated to Map rather than deleted? Or is Map somewhere else?